src/core/server/saved_objects/mappings/lib/get_root_properties_objects.test.ts

```diff
@@ -34,7 +34,7 @@ test(`returns single object with properties`, () => {
         properties: {},
       },
     },
-  };
+  } as const;
```

Comment:
Are these required for TypeScript to successfully compile?

Reply:
Yes. See type error when compile. For example, if remove const, the following type errors are shown. 
```
const mappings: {
    properties: {
        foo: {
            type: string;
        };
    };
}
Argument of type '{ properties: { foo: { type: string; }; }; }' is not assignable to parameter of type 'IndexMapping'.
  Types of property 'properties' are incompatible.
    Type '{ foo: { type: string; }; }' is not assignable to type 'SavedObjectsMappingProperties'.
      Property 'foo' is incompatible with index signature.
        Type '{ type: string; }' is not assignable to type 'SavedObjectsFieldMapping'.
          Type '{ type: string; }' is not assignable to type 'MappingGenericProperty & { dynamic?: false | "strict" | undefined; }'.
            Type '{ type: string; }' is missing the following properties from type 'MappingGenericProperty': analyzer, boost, fielddata, ignore_malformed, and 7 more.
```